feelpp/contrib/ginac/ginac/CMakeLists.txt

```diff
@@ -154,10 +154,10 @@ add_definitions(-DLIBEXECDIR="${LIBEXECDIR}/")
 set_target_properties(feelpp_ginac PROPERTIES
 	SOVERSION ${ginaclib_soversion}
 	VERSION ${ginaclib_version})
-target_link_libraries(feelpp_ginac ${CLN_LIBRARIES} ${CMAKE_DL_LIBS} ${Boost_MPI_LIBRARY} ${Boost_SERIALIZATION_LIBRARY} ${Boost_FILESYSTEM_LIBRARY} ${Boost_SYSTEM_LIBRARY} ${MPI_LIBRARIES})
+target_link_libraries(feelpp_ginac cln::cln ${CMAKE_DL_LIBS} ${Boost_MPI_LIBRARY} ${Boost_SERIALIZATION_LIBRARY} ${Boost_FILESYSTEM_LIBRARY} ${Boost_SYSTEM_LIBRARY} ${MPI_LIBRARIES})
```

Comment:
cln::cln exist with external dependency? (in FindCLN)

Reply:
yes this is from cln

Reply:
I had to retrieve cln to fix the arm code generation

Reply:
an alternative is to compile from source cln and use the recent git version